blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -0,0 +1,130 @@
+---
+path: /blog/gather-metaverse-bump-each-other-2021/
+date: 2021-12-21
+title: How we joined the Metaverse with Gather 
+featuredImage: images/gather/teaser.png
+seoMetaText: A few weeks ago, we introduced Gather at Satellytes to fight Zoom Fatigue. Soon after, we saw the magic of the Metaverse happening.
+teaserText: A few weeks ago, we introduced Gather at Satellytes to make our daily remote life more pleasant and to fight Zoom Fatigue. Soon after, we saw the magic of the Metaverse happening.
+attribution:
+    creator: Made with Gather Tiles
+    source: https://github.com/gathertown/mapmaking
+author: Georgios Kaleadis
+authorSummary: CTO at Satellytes
+---
+
+## Introduction
+A few weeks ago, we introduced [Gather](https://www.gather.town/) at Satellytes to make our daily remote life more pleasant and to fight [Zoom Fatigue](https://en.wikipedia.org/wiki/Zoom_fatigue). Gather describes itself as a **_"platform that combines a 2D game-like interface with video-chat"_**. 
+
+![Official gather screenshot with people hanging around](images/gather/gather-example-official.png)
+_Official gather screenshot with people hanging around_
+
+It might look like a game, but it's only the optics. Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```

Comment:
```suggestion
At first glance it might look like a game. But that's only the visual presentation chosen by gather (although you can run go-karts, if you like to 😄). The core use-cases are actually all focused around the office live. 

Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```
```suggestion
It might look like a game, but it's only the optics. Only people you invite can access your world. You pick an avatar and walk around a virtual office. Once you bump into a colleague, a small video window opens near your avatar, and you can start talking. When you walk away from a conversation, the video and voices diminish until the chat closes. When you focus a different window, while not being in a conversation, your video and audio are always muted, but people can still try to approach.
```

Reply:
what happened with your first suggestion? I see a lot of html mixed in ?